src/tests/ftest/control/dmg_telemetry_net.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python3
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+    SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import threading
+import time
+
+from re import search
+from mdtest_test_base import MdtestBase
+from telemetry_test_base import TestWithTelemetry
+from general_utils import DaosTestError, pcmd
+from command_utils import CommandFailure
+
+# pylint: disable=too-few-public-methods,too-many-ancestors
+# pylint: disable=attribute-defined-outside-init
+class TestWithTelemetryNet(MdtestBase, TestWithTelemetry):
+    """Rebuild test cases featuring mdtest.
+
+    This class contains tests for pool rebuild that feature I/O going on
+    during the rebuild using IOR.
+
+    :avocado: recursive
+    """
+
+    def pre_tear_down(self):
+        """Tear down any test-specific steps prior to running tearDown().
+
+        Returns:
+            list: a list of error strings to report after all tear down
+            steps have been attempted
+
+        """
+        # kill any hanging mdtest processes
+        ret = []
+
+        # DEBUGGING
+        pcmd(self.hostlist_servers, "ps aux")
+
+        cmd = "pkill -9 mdtest"
+        for retry in range(0, 5):
+            ret_codes = pcmd(self.hostlist_servers, cmd)
+            if len(ret_codes) > 1 or 0 not in ret_codes:
+                self.log.info("Retry #{}: Not all '{}' commands in pcmd "
+                              "returned 0.".format(retry, cmd))
+
+        # Ignore errors cleaning up mdtest
+        return []
+
+    def tearDown(self):
+        try:
+            super().tearDown()
+        except (CommandFailure, DaosTestError) as excep:
+            self.log.info("{}: tearDown threw an exception "
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions